### PR TITLE
CLOUD-317 improve pr reporting

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| storage | passed |
| init-deploy | passed |
| demand-backup | passed |
| monitoring | failed |
| limits | passed |
| scheduled-backup | passed |
| scaling | passed |
| operator-self-healing | passed |

File: Jenkinsfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ void CreateCluster(String CLUSTER_PREFIX) {
             source $HOME/google-cloud-sdk/path.bash.inc
             gcloud auth activate-service-account --key-file $CLIENT_SECRET_FILE
             gcloud config set project $GCP_PROJECT
-            gcloud container clusters create --zone us-central1-a $CLUSTER_NAME-${CLUSTER_PREFIX} --cluster-version 1.11 --machine-type n1-standard-4 --preemptible --num-nodes=3 --network=jenkins-vpc --subnetwork=jenkins-${CLUSTER_PREFIX}
+            gcloud container clusters create --zone us-central1-a $CLUSTER_NAME-${CLUSTER_PREFIX} --cluster-version 1.12 --machine-type n1-standard-4 --preemptible --num-nodes=3 --network=jenkins-vpc --subnetwork=jenkins-${CLUSTER_PREFIX}
```
Comment (delgod) [inline Jenkinsfile]:
```suggestion
            gcloud container clusters create --zone us-central1-a $CLUSTER_NAME-${CLUSTER_PREFIX} --cluster-version 1.14 --machine-type n1-standard-4 --preemptible --num-nodes=3 --network=jenkins-vpc --subnetwork=jenkins-${CLUSTER_PREFIX}
```

Comment (hors) [reply Jenkinsfile]: I will do it via separate review.

Comment (hors) [reply Jenkinsfile]: I will do it via separate review.

**Pull request merged**